lib/Minz/Mailer.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+
+require LIB_PATH . '/PHPMailer/PHPMailer.php';
+require LIB_PATH . '/PHPMailer/Exception.php';
+require LIB_PATH . '/PHPMailer/SMTP.php';
+
+/**
+ * Allow to send emails.
+ *
+ * The Minz_Mailer class must be inherited by classes under app/Mailers.
+ * They work similarly to the ActionControllers in the way they have a view to
+ * which you can pass params (eg. $this->view->foo = 'bar').
+ *
+ * The view file is not determined automatically, so you have to select one
+ * with, for instance:
+ *
+ * ```
+ * $this->view->_path('user_mailer/email_need_validation.txt')
+ * ```
+ *
+ * Minz_Mailer uses the PHPMailer library under the hood. The latter requires
+ * PHP >= 5.5 to work. If you instantiate a Minz_Mailer with PHP < 5.5, a
+ * warning will be logged.
+ *
+ * The email is sent by calling the `mail` method.
+ */
+class Minz_Mailer {
+	/**
+	 * The view attached to the mailer.
+	 * You should set its file with `$this->view->_path($path)`
+	 *
+	 * @var Minz_View
+	 */
+	protected $view;
+
+	/**
+	 * Constructor.
+	 *
+	 * If PHP version is < 5.5, a warning is logged.
+	 */
+	public function __construct () {
+		if (version_compare(PHP_VERSION, '5.5') < 0) {
+			Minz_Log::warning('Minz_Mailer cannot be used with a version of PHP < 5.5.');
+		}
```

Comment:
Should we maybe return and fail here?

Reply:
Well, in fact I think it *might* work if PHPMailer is configured to use the `mail` function. I just don't want to support this case officially, but I'm ok to let a possibility to admins to force the feature despite the warnings.